geomesa-convert/geomesa-convert-common/src/main/scala/org/locationtech/geomesa/convert/CompositeConverter.scala

```diff
@@ -54,7 +52,7 @@ class CompositeConverter[I](val targetSFT: SimpleFeatureType,
 
   implicit val ec = new EvaluationContext(Map(), Array())
   def processIfValid(input: I, pred: Predicate, conv: SimpleFeatureConverter[I]) =
-    if(pred.eval(input)) conv.processSingleInput(input) else None
+    Try { pred.eval(input) }.toOption.flatMap { v => if(v) conv.processSingleInput(input) else None }
```

Comment:
catch the exception you expect to occur.. toOption will turn any Failure into a None. For example you probably don't want to turn a NPE into a None.. you want to throw that particular exception.